stdlib/decimal.pyi

```diff
@@ -66,6 +70,10 @@ class FloatOperation(DecimalException, TypeError): ...
 
 class Decimal:
     def __new__(cls, value: _DecimalNew = "0", context: Context | None = None) -> Self: ...
+    if sys.version_info >= (3, 14):
+        @classmethod
+        def from_number(cls, number: float, /) -> Self: ...
```

Comment:
```suggestion
        def from_number(cls, number: Decimal | float, /) -> Self: ...
```
See `PyDecType_FromNumberExact` in the C code

Reply:
thank you